File: 프로그래머스/lv0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.py
def solution(my_string):
    buffer = ""
    answer = []
    for i in my_string:
        if i.isdigit():
            buffer += i
        else:
            if buffer:
                answer.append(int(buffer))
                buffer = ""
    if buffer:
        answer.append(int(buffer))
    return sum(answer)

File: 프로그래머스/lv0/120864. 숨어있는 숫자의 덧셈 （2）/숨어있는 숫자의 덧셈 （2）.java
class Solution {
    public int solution(String my_string) {
        int sum = 0;
        String s = my_string.replaceAll("[a-zA-Z]", " ");
        String[] split = s.split(" ");
        for (String i : split) {
            if (!i.equals("")) {
                sum += Integer.valueOf(i);
            }
        }
        return sum;
    }
}